tools/modules/software/module_postgres.sh

```
@@ -0,0 +1,103 @@
+	["module_postgres,arch"]="x86-64 arm64"
+)
+
+#
+# Module postgres
+#
+function module_postgres () {
+	local title="postgres"
+	local condition=$(which "$title" 2>/dev/null)
+
+	if pkg_installed docker-ce; then
+		local container=$(docker container ls -a | mawk '/postgres?( |$)/{print $1}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)